Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Multistage docker build, requires docker 17.05
 
 # builder stage
-FROM ubuntu:16.04 as builder
+FROM debian:jessie as builder
```

Comment:
Why ?

Reply:
Checked under Ubuntu. In next commit revert to 16.04